src/dirbrowserformaction.cpp

```diff
@@ -295,24 +284,20 @@ void DirBrowserFormAction::init()
 
 	set_value("fileprompt", _("Directory: "));
 
-	if (dir == "") {
-		std::string save_path = cfg->get_configvalue("save-path");
+	const std::string save_path = cfg->get_configvalue("save-path");
 
-		LOG(Level::DEBUG,
-			"view::dirbrowser: save-path is '%s'",
-			save_path);
-
-		dir = save_path;
-	}
+	LOG(Level::DEBUG,
+		"view::dirbrowser: save-path is '%s'",
+		save_path);
 
-	int status = ::chdir(dir.c_str());
-	LOG(Level::DEBUG, "view::dirbrowser: chdir(%s) = %i", dir, status);
+	int status = ::chdir(save_path.c_str());
```

Comment:
I reckon this can now be `const` as well.

Reply:
Thanks, suggestion squashed into https://github.com/newsboat/newsboat/pull/1421/commits/0f31b9da5dd8ded7a7178c1986867ceb5af802b1.